Abubakarharuna10.md

```diff
@@ -0,0 +1,41 @@
+---
+name: "Abubakarharuna10"
+
+project: "Artificial Rescuer"
+
+Slack Username "@abubakarharuna"
+
+---
+
+## Description
+
+With the increasing threat of climate change, greater populations and consequently, more extreme environmental events
+the need for better emergency rescue services is paramount
+As time moves forward, natural disasters will continue to wreak havoc on cities and counties around the world, 
+causing many people to die. Bolstering the power of search and rescue teams with autonomous UAVs that can identify people
+in disaster relief zones could significantly reduce the impact tornadoes, earthquakes, tsunamis, floods, etc...
+My solution to strengthening search and rescue operations is to outfit an autonomous unmanned aerial vehicle (UAV) 
+with a computer vision system that detects the location of people as the vehicle flies over ground. The powerful neural-network capabilities
+of the Jetson Nano Dev Kit will enable fast computer vision algorithms to achieve this task. Joseph Redmon's YOLOv3 algorithm will be used 
+to do the actual object-detection (people) in the camera's view While the UAV is flying a waypoint mission using ArduPilot, PX4, or any other autonomous flight control stack, the absolute location of people in the camera view can be calculated based on the altitude, orientation, and GPS location of the UAV.
+
+## Plan
+1) Having requirements to use in this project.
+2) Hardware components,
+  -NVIDIA Jetson Nano Developer Kit
+  -Edimax EW-7811Un 150Mbps 11n Wi-Fi USB Adapter
+  -CanaKit 5V 2.5A Power Supply
+  -Raspberry Pi Camera Module V2
+  -Anker PowerCore 5000
+   -PX4 Pixhawk
+3) Implimentation
```

Comment:
In your plan, could you describe how the components fit together and what steps you would take to do that? Ideally, someone reading this document would be able to imagine what the final product would look like.

Reply:
I update it again!